srcpkgs/bankstown/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'bankstown'
+pkgname=bankstown
+version=1.1.0
+revision=1
+build_style=cargo
+makedepends="lv2"
+depends="lv2>=0.6.0"
```

Comment:
is this >= requirement necessary?

Reply:
This looks like a misunderstanding: the package uses cargo's lv2 crates, and it should not depend on lv2 at all.